bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -577,32 +577,32 @@ The following icons are used by original Netatmo web app:
 
 ## Modules
 
-- http://my.netatmo.com/img/my/app/module_int.png
-- http://my.netatmo.com/img/my/app/module_ext.png
-- http://my.netatmo.com/img/my/app/module_rain.png
+- http://my.netatmo.com/images/my/app/module_int.png
```

Comment:
Can you please change all links to HTTPS? Thanks.

Reply:
Done